contracts/modules/cover/Cover.sol

```diff
@@ -216,7 +216,7 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
 
     _updateTotalActiveCoverAmount(
       params.coverAsset,
-      coverAmountInCoverAsset,
+      params.amount, // TODO: check this change
```

Comment:
This isn't accurate because the `coverAmountInCoverAsset` will likely be slightly more due to some rounding up.